src/Modules/Family/PrimaryInfoFormComponent.js

```diff
@@ -1,229 +1,79 @@
-                <div className="form-group">
-                    <label>Middle Name</label>
-                    <input type="text" className="form-control" onChange={buildNameForm} defaultValue={middle_name_form}
-                           name="middle_name" id="middle_name" />
-                </div>
-                <div className="form-group">
-                    <label>Last Name</label>
-                    <input type="text" className="form-control" onChange={buildNameForm} name="last_name" id="last_name"
-                           onBlur={handleErrors} />
-                </div>
-
-                <div className="form-group">
-                    <label>Suffix</label>
-                    <select  id="suffix" name="Suffix" className="form-control"  onChange={buildNameForm}>
-                        <option>Jr</option>
-                        <option>Sr</option>
-                    </select>
-                </div>
-                <div className="form-group">
-                    <label>Date of Birth</label>
-                    <input type="date" className="form-control"  name="dob" id="dob"/>
-                </div>
-                <div className="form-group">
-                    <label>Head of Household</label>
-                    <select id="hoh" name="hoh"  onChange={buildNameForm}  className="form-control">
-                        <option>Yes</option>
-                        <option>No</option>
-                    </select>
-                </div>
-
-                <div className="form-group">
-                    <label>Phone Number</label>
-                    <input type="text" className="form-control" onChange={buildNameForm} name="phone_number" id="phone_number"
-                           onBlur={handleErrors}/>
-                </div>
-
-                <div className="form-group">
-                    <label className="custom-checkbox">
-                        <input type="checkbox" className="checkbox"  name="phone_number_checkbox"  id ="phone_number_checkbox" onChange={buildNameForm}/>
-                        <span>No Phone Available</span>
-                    </label>
-                </div>
-                <div className="form-group">
-                    <label>Email Address</label>
-                    <input type="text" className="form-control" onChange={buildNameForm} name="email" id="email"/>
-                    <small className="text-muted">
-                        No Email? <a href="">Get one free from Google.</a>
-                    </small>
-                </div>
-                <div className="form-group">
-                    <label>Communication Preference</label>
-                    <select  id='communication_preference' name='communication_preference' onChange={buildNameForm} className="form-control">
-                        <option>Email</option>
-                        <option>Email</option>
-                    </select>
-
-            </div>
-        </div>
-    )
-});
```

Comment:
Check the diff in this component. `react-hook-form` reduces a ton code that is no longer needed to be maintained.